kernel/kernel-api/src/main/java/io/delta/kernel/utils/VectorUtils.java

```diff
@@ -30,7 +30,8 @@ public final class VectorUtils {
     private VectorUtils() {}
```

Comment:
Can we consolidate the `toJavaMap(MapValue)` and `toMapValue(java.util.Map` here? Similar for the array.

Reply:
Responding to the (I think) 3 comments related to this here.

We need to expose java list/maps as MapValue/ArrayValue (and thus as ColumnVectors) in a couple of places

In kernelApi:
- (1) JsonHandlerTestImpl

In kernelDefaults:
- (2) DefaultJsonHandlerImpl (in DefaultJsonRow) -- very similar pattern to the first
- (3) Tests (once in ExpressionEvaluatorSuite)

I can add a test utility to do this to cover (1) and (3) but I don't think we want to add this to `VectorUtils` as we shouldn't be creating ColumnVectors there?

Reply:
Agree, it should be outside the `VectorUtils` somewhere in defaults module. I think we may have to remove `DefaultJsonHandleImpl` and make it a proper mock. I will create a task for this.

Reply:
How about add utility method for defaults and it can be used in default prod code and test code.

For the `JsonHandlerTestImpl`, I am actually thinking of removing it and make it a proper mock.

Reply:
Added a `DefaultGenericVector` and use that in `DefaultJsonRow` and in a test utility I added that converts a `Seq[Map[..,..]]`to a CV. Can't really combine more than that since in `DefaultJsonRow` we can build the keys/values separately and don't start with a `Map`